sharedPrompts/src/main/java/org/example/sharedprompts/ENV_VARIABLES.md

```
@@ -0,0 +1,440 @@
+## 23. 쿠키 보안
+
+| 변수명 | 용도 설명 | 예시 값 |
+|--------|----------|---------|
+| auth.cookie.secure | HTTPS 전용 쿠키 (prod: true 권장) | false |
+
+---
+
+## 24. 비밀번호 정책
+
+| 변수명 | 용도 설명 | 예시 값 |
+|--------|----------|---------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)